publish/action.yml

```diff
@@ -35,3 +35,10 @@ runs:
     - name: Build package
       shell: bash
       run: mvn -B -s ${{ inputs.mvn_settings_filepath }} clean deploy
+      id: artifact_build
+
+    - name: Direct link location of the Artifact
+      shell: bash
+      run: |
+        EXTRACTED_NEXUS_ARTIFACT_URL=`grep -e "Uploaded.*\.jar" ${{steps.artifact_build.outputs}} | grep -v sources | sed -e 's/.*\(https.*.jar\).*/\1/g'` 
+        echo "::notice title=Artifact location (require VPN)::Artifact has been uploaded to: $EXTRACTED_NEXUS_ARTIFACT_URL"
```

Comment:
What URLs are we talking about ? If it's on nexus, there's no need for the VPN.

Since there's only one artifact, I'd also rename this to `Snapshot module location`instead of `Artifact location (require VPN)`

Reply:
Fixed, yes we talking about nexus, thanks